src/linker/Linker/LinkerAttributesInformation.cs

```diff
@@ -124,18 +124,9 @@ public static LinkerAttributesInformation Create (LinkContext context, ICustomAt
 			return null;
 		}
 
-		static RemoveAttributeInstancesAttribute? BuildRemoveAttributeInstancesAttribute (LinkContext context, TypeDefinition attributeContext, CustomAttribute ca)
+		static RemoveAttributeInstancesAttribute? BuildRemoveAttributeInstancesAttribute (LinkContext _, TypeDefinition __, CustomAttribute ca)
```

Comment:
Instead of calling the method you could just do `new RemoveAttributeInstancesAttribute (ca.ConstructorArguments);` and delete the method, also flag the diagnostic as _unused_ in the [DiagnosticId.cs](https://github.com/dotnet/linker/blob/main/src/ILLink.Shared/DiagnosticId.cs) file.

Reply:
My bad there was more than one instance of the diagnostic creation. Sorry